aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -162,6 +164,13 @@ private FileIO initializeFileIO(Map<String, String> properties) {
     }
   }
 
+  @VisibleForTesting
+  void initialize(String name, String path, AwsProperties properties, GlueClient client,
```

Comment:
It isn't clear why this method is needed. @SinghAsDev, is this necessary?

Reply:
This shouldn't be necessary, it is just used in test to check properties map is not empty when catalog properties is initialised.